site/src/api/typesGenerated.ts

```diff
@@ -646,6 +646,12 @@ export interface DatabaseReport extends BaseReport {
 	readonly threshold_ms: number;
 }
 
+// From codersdk/database.go
+export const DefaultIdleConns = 3;
+
+// From codersdk/database.go
+export const DefaultMaxConns = 10;
+
```

Comment:
🤔 I'm not sure the API needs to be aware of this? I think there's an annotation you can set on these constants to exclude them.